release-notes/opensearch-release-notes-2.0.1.md

```diff
@@ -0,0 +1,31 @@
+# OpenSearch and OpenSearch Dashboards 2.0.1 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.1 includes the following bug fixes, documentation, and maintenance updates.
```

Comment:
Remove "documentation" from this sentence.

Reply:
Actually there is one from alerting dashboards. Changed the font size for the `Documentation` below.